archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/ClassesThat.java

```diff
@@ -413,6 +419,9 @@
      * Matches classes that implement a certain interface matching the given predicate. For example, a call with
      * {@link Predicates#name(String)} would be equivalent to
      * {@link #implement(String)}, but the approach is a lot more generic.
+     * Note that this only matches non-interface {@link JavaClass classes} that implement an interface matching the {@code predicate}
+     * (compare {@link JavaClass.Predicates#implement(Class)}.
```

Comment:
```suggestion
     * (compare {@link JavaClass.Predicates#implement(DescribedPredicate)}.
```

Reply:
As described above, I left these, because it's really meant as a "compare", i.e. see the explanation of the semantics there, not see the equivalent method there. I hope it's still clear, I just feel that jumping to a method that simply says "this is the same as $otherMethod, see there" is also somewhat frustrating :wink: